stdlib/InteractiveUtils/src/InteractiveUtils.jl

```diff
@@ -308,36 +307,12 @@ function dumpsubtypes(io::IO, x::DataType, m::Module, n::Int, indent)
     nothing
 end
 
-const Distributed_modref = Ref{Module}()
-
-"""
-    peakflops(n::Integer=2000; parallel::Bool=false)
-
-`peakflops` computes the peak flop rate of the computer by using double precision
-[`gemm!`](@ref LinearAlgebra.BLAS.gemm!). By default, if no arguments are specified, it
-multiplies a matrix of size `n x n`, where `n = 2000`. If the underlying BLAS is using
-multiple threads, higher flop rates are realized. The number of BLAS threads can be set with
-[`BLAS.set_num_threads(n)`](@ref).
-
-If the keyword argument `parallel` is set to `true`, `peakflops` is run in parallel on all
-the worker processors. The flop rate of the entire parallel computer is returned. When
-running in parallel, only 1 BLAS thread is used. The argument `n` still refers to the size
-of the problem that is solved on each processor.
-"""
```

Comment:
I think the docstring needs to stay on the exported function for `?peakflops` to work.

Reply:
Good catch, I'll include a note that this function will be moved in the future too.